hal/tests/TESTS/mbed_hal/gpio/main.cpp

```diff
@@ -30,13 +30,15 @@ static void gpio_nc_test()
     gpio_init(&nc_obj, NC);
     TEST_ASSERT_FALSE(gpio_is_connected(&nc_obj));
 
+#ifdef LED1
```

Comment:
Would it make sense to also add LED2 and LED3
I think we should have enough with 3 but happy to increase to 5!

Reply:
In fact, I think this test has to be removed, and replaced by the new one from #13913

Reply:
Indeed, I agree.
Please make the changes and let me know once it's ready for review.

Reply:
Done